### ISSUE TITLE
Failure to enumerate/install user fonts if the user's fonts folder is missing

### ISSUE THREAD

**Issue by iwd-sys** [ralish/PSWinGlue#8]
Hi,

If the user mode font folder is empty, then it is impossible to install a font in user mode.

When this check is done:

```
if ($FontHash.Hash -notin $InstalledFonts.Hash)
```

`$InstalledFonts` is not defined when there are no fonts currently installed, hence the `Hash` property fails.

Thanks.

**Comment by ralish** [comment]
I'm having trouble reproducing this. Can you please provide the following:
- Windows version  
  `[Environment]::OSVersion.Version`
- PowerShell version  
  `$PSVersionTable.PSVersion`
- Module version  
  `Get-Module -Name PSWinGlue -ListAvailable | select -p Version`
- Full command you're running and resulting output (esp. errors/exceptions)

Thanks.

**Comment by ralish** [comment]
@iwd-sys Can you provide the above and I'll get this fixed?

**Comment by iwd-sys** [comment]
Hi, sorry, didn't see the notification.

- Windows Server 2019
- 5.1.17763 2931
- v0.6.4

I don't recall the exact error message. But I can share my hotfix:

Before:

```powershell
if ($FontHash.Hash -notin $InstalledFonts.Hash) {
    $InstallFonts.Add($Font)
} else {
    Write-Verbose -Message ('Font is already installed: {0}' -f $Font.Name)
}
```

After:

```powershell
if ($InstalledFonts -eq $null) {
    $InstallFonts.Add($Font)
} elseif ($FontHash.Hash -notin $InstalledFonts.Hash) {
    $InstallFonts.Add($Font)
} else {
    Write-Verbose -Message ('Font is already installed: {0}' -f $Font.Name)
}
```

**Comment by felipecrs** [comment]
I'm facing a very similar error on the working computers of my colleagues. It turned out that they did not have the Fonts folder under AppData and manually creating it makes the Install-Font.ps1 script work.

However, I cannot reproduce the issue on my personal computer too.

**Comment by felipecrs** [comment]
Nevermind, actually it's very simple to reproduce. Simply remove the folder:

C:\Users\<user>\AppData\Local\Microsoft\Windows\Fonts

And the script fails with:

```
Unable to enumerate user fonts folder: C:\Users\felip\AppData\Local\Microsoft\Windows\Fonts
At \\wsl.localhost\Ubuntu\home\felipecrs\.dotfiles\external-scripts\Install-Font.ps1:117 char:9
+         throw 'Unable to enumerate {0} fonts folder: {1}' -f $Scope.T ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Unable to enume...t\Windows\Fonts:String) [], RuntimeException
    + FullyQualifiedErrorId : Unable to enumerate user fonts folder: C:\Users\felip\AppData\Local\Microsoft\Windows\Fonts
```

**Comment by ralish** [comment]
Thanks all. I've implemented a fix which resolves the issue in both `Get-Fonts` and `Install-Font`. The `Uninstall-Font` command is unaffected. The fix is in the v0.6.6 release I just published.
